### PR TITLE
Fix server configuration

### PR DESCRIPTION
**Pull request by utiq**
Putting back webserver container in the docker-compose.yml file
Deleted the pusher configuration since another one that it's tested was created in this PR that I need to merge https://github.com/adamlerer/webDiplomacy/pull/137/files

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -51,17 +51,22 @@ services:
 
   clickhouse:
     image: "yandex/clickhouse-server:latest"
-
-  # This is necessary for doing live React dev for the new board, but isn't needed after doing a build
-  # if the nginx config is changed to not redirect /beta/ to port 3000:
+  webserver:
```
Comment (adamlerer) [inline docker-compose.yml:54]: Put a comment here explaining why this is necessary so @kestasjk doesn't remove it again.

File: docker-compose.yml [hunk 2]
```diff
@@ -51,17 +51,22 @@ services:
...
+      - "${WEBDIP_PORT:-43000}:80"
   beta:
     image: "node:16.15.1-alpine3.14"
     container_name: webdiplomacy-beta
     working_dir: /application
     volumes:
       - "./beta-src:/application"
     ports:
-      - "3000"
+      - "3000:3000" # This is necessary for doing live React dev for the new board, but isn't needed after doing a build. If the nginx config is changed to not redirect /beta/ to port 3000:
```
Comment (adamlerer) [inline docker-compose.yml:69]: I'm not understanding the second sentence here, is there a clause missing?

File: config.sample.php [hunk 3]
```diff
@@ -152,18 +152,6 @@ class Config
 	 */
 	public static $memcachedPort='11211';
```
Comment (adamlerer) [inline config.sample.php:154]: Did you mean to remove the pusher stuff?

**Review by adamlerer — COMMENTED** [review]
Can you write a couple sentences PR comment explaining what you're doing with the configuration? Adding back `webserver` is great; I'm not sure what the other changes are about.

File: config.sample.php (hunk 3 above)
Comment (utiq) [reply config.sample.php:154]: yes, cuz that config was not ok. I made a config in the PR I created
https://github.com/adamlerer/webDiplomacy/blob/63099958d2a31d08204c2b83261d32e9e106ebde/config.sample.php#L489-L519

File: docker-compose.yml (hunk 2 above)
Comment (utiq) [reply docker-compose.yml]: I just put it back to where it was

File: docker-compose.yml (hunk 1 above)
Comment (utiq) [reply docker-compose.yml:56]: I put it back to where it was

**Comment by utiq** [comment]
> Can you write a couple sentences PR comment explaining what you're doing with the configuration? Adding back `webserver` is great; I'm not sure what the other changes are about.

done

**Pull request merged**